actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -291,7 +298,7 @@ private void updateServerNodes()
 
     private Task<NodeAddress> locateActiveActor(final RemoteReference<?> actorReference)
     {
-        final NodeAddress address = localAddressCache.get(actorReference);
+        final NodeAddress address = localAddressCache.asMap().get(actorReference);
```

Comment:
yep, I had actually done a little more work on it before I realised this pull request was already open.. :man_facepalming: https://github.com/orbit/orbit/pull/252